src/databases/PVLD/PVLD.xml

```diff
@@ -1,16 +1,18 @@
 <?xml version="1.0"?>
-  <Plugin name="PVLD" type="database" label="Partitioned Velodyne plot file" version="1" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" dbtype="STMD" haswriter="false" hasoptions="false" filePatternsStrict="false" opensWholeDirectory="false">
+  <Plugin name="PVLD" type="database" label="Partitioned Velodyne plot file" version="5" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" dbtype="STMD" haswriter="false" hasoptions="false" filePatternsStrict="false" opensWholeDirectory="false">
     <FilePatterns>
       *.pvld *.vld
     </FilePatterns>
     <CXXFLAGS> 
       ${HDF5_INCLUDE_DIR}
+      -std=c++1y
```

Comment:
Is the -std flag strictly necessary? Was there a specific os/compiler that necessitated its addition?